### PR TITLE
Add cloudflare-go runner handler.

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

File: impl-endpoints/cloudflare-go/runner.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+func main() {
+	testCase := flag.String("test", "", "Test case identifier")
+	role := flag.String("role", "client", "Role string (client or server)")
+	flag.Parse()
+
+	handler, ok := testCaseHandlers[*testCase]
+	if !ok {
+		// Skip this test case
+		os.Exit(-2)
```
Comment (chris-wood) [inline impl-endpoints/cloudflare-go/runner.go:53]: @xvzcf this return code was arbitrary -- we can change it to whatever makes sense!

File: impl-endpoints/cloudflare-go/run_endpoint.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ if [ "$ROLE" = "client" ]; then
     echo "Running Cloudflare-Go client."
     echo "Client params: $SERVER_PARAMS"
     echo "Test case: $TESTCASE"
-    /cf-go/bin/go run /client.go
+    /cf-go/bin/go run /runner.go -role=client -test=${TESTCASE}
```
Comment (xvzcf) [inline impl-endpoints/cloudflare-go/run_endpoint.sh:11]: Hmmm, perhaps this file can be removed and we can just pass arguments to the runner through Dockerfile `ARG`s. However, setting up Wireshark and ns-3 might require a script like this. What do you think?

File: Makefile [hunk 3]
```diff
@@ -8,8 +8,11 @@ cert-tool: $(CERT_TOOL_FILES)
...
-	cert-tool -cert-out certs/server.cert -key-out certs/server.key -dc-capable server
+	./cert-tool -CA
+	./cert-tool -cert-out certs/server.cert -key-out certs/server.key -dc-capable server
 
 dc: certs/server.cert certs/server.key cert-tool
-	cert-tool -cert-in certs/server.cert -key-in certs/server.key -generate-dc -dc-out certs/dc.txt
+	./cert-tool -cert-in certs/server.cert -key-in certs/server.key -generate-dc -dc-out certs/dc.txt
+
+clean:
+	docker system prune -a
```
Comment (xvzcf) [inline Makefile:18]: Could you replace this with [docker builder prune](https://docs.docker.com/engine/reference/commandline/builder_prune/)? Just found out about this option.

File: impl-endpoints/cloudflare-go/run_endpoint.sh (hunk 2 above)
Comment (chris-wood) [reply impl-endpoints/cloudflare-go/run_endpoint.sh:11]: I'm not sure how to do that, so maybe we can just merge this and you could tidy it up after?

**Pull request merged**